lib/ddr/models/has_content.rb

```diff
@@ -42,6 +42,8 @@ def master_file?
 
         around_save :update_derivatives, if: :content_changed?
 
+        around_save :characterize_file, if: :content_changed?
```

Comment:
We'll want to remember to add a condition to skip the callback if the fits datastream content has changed -- when/if we submit FITS output with batch objects.